src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -103,6 +108,36 @@ ContextAttribute::ContextAttribute(ContextAttribute* caP)
 
 /* ****************************************************************************
 *
+* ContextAttribute::ContextAttribute -
+*/
+ContextAttribute::ContextAttribute
+(
+  const std::string&  _name,
+  const std::string&  _type,
+  const char*         _value,
+  bool                _found
+)
+{
+  LM_T(LmtClone, ("Creating a string ContextAttribute '%s':'%s':'%s', setting its compound to NULL",
+                  _name.c_str(),
+                  _type.c_str(),
+                  _value));
+
+  name                  = _name;
+  type                  = _type;
+  stringValue           = std::string(_value);
+  valueType             = ValueTypeString;
+  compoundValueP        = NULL;
+  found                 = _found;
+
+  providingApplication.set("");
+  providingApplication.setFormat(NOFORMAT);
+}
```

Comment:
We need to verify that this constructor makes sense (having one with "std::string" instead of "char*")